identity/src/main/java/com/android/identity/Utility.java

```diff
@@ -18,19 +18,32 @@
 
```

Comment:
I was going to ask if we move this entire file to `android/util/AndroidUtil.java` because I think only Android stuff currently depends on it... but I think we can also just do that in a later PR.

Reply:
ack